src/js/components/PageHeader/stories/MultipleActions.js

```diff
@@ -0,0 +1,81 @@
+import React, { useContext } from 'react';
+
+import {
+  Anchor,
+  Box,
+  Button,
+  Menu,
+  PageHeader,
+  Page,
+  PageContent,
+  ResponsiveContext,
+} from 'grommet';
+import { More } from 'grommet-icons';
+
+const actions = [
+  {
+    label: 'Get Started',
+    primary: true,
+  },
+  {
+    label: 'Follow',
+    secondary: true,
+  },
+  {
+    label: 'File Issue',
+    secondary: true,
+  },
+];
+
+const controls = {
+  small: (
```

Comment:
one small thing sorry do we need a `xsmall` here or are we not worried about that size for the hpe theme version

Reply:
Since grommet only has `small - large` breakpoints, I don't think we would include it here but for an example on DS site we would implement with `xsmall`

Reply:
thats fine I just saw when I switched the theme on storybook